flink-runtime/src/main/java/org/apache/flink/runtime/state/KeyedStateHandle.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.state;
+
+/**
+ * A snapshot in keyed streams. Each {@code KeyedStateHandle} contains the
```

Comment:
A state handle typically means a pointer to some state. It will mostly be used for snapshots of state backends, but conceptional this i a different level of abstraction. I suggest to slightly modify this comment, moving away from the concrete implementations of snapshots, to the concept of having a pointer to serialized keyed state.